webapp/packages/plugin-object-viewer/src/ObjectPropertiesPage/ObjectPropertyTable/Table/Table.tsx

```diff
@@ -125,28 +65,70 @@ export const Table = observer<TableProps>(function Table({ objects, hasNextPage,
     return null;
   }
 
+  function getCell(rowIdx: number, colIdx: number) {
+    colIdx--;
```

Comment:
shouldn't we move this mutation to the DataGrid props 
so `getCell={(r, c) => getCell(r, c--)}`?